Kudu.Core/Environment.cs

```diff
@@ -266,6 +270,18 @@ public string SiteExtensionSettingsPath
             get { return _siteExtensionSettingsPath; }
         }
 
+        public bool AlwaysOnEnabled
+        {
+            get { return _alwaysOnEnabled; }
+        }
+
+        internal bool IsAlwaysOnEnabled()
+        {
+            string value = System.Environment.GetEnvironmentVariable(WebsiteSCMAlwaysOnEnabledKey);
+
+            return string.Compare("1", value) == 0;
```

Comment:
Mainly I just want to hold on checking this in until the Antares change is made and signed off on, so there is no churn in the env name. That's all. Once I get the review out for that and the name is settled, this can go in. The warning will then only be active once the Antares change goes live.

I don't want to emit this warning until then, so I don't want to default to true in absence of the env value, since them message says "it looks like you don't have Always On". For people who do, I don't want to write that.